### DISCUSSION TITLE
Struggles with no backspace on number layer, and ()

### DISCUSSION THREAD

**Discussion by jasonish** [manna-harbour/miryoku#138] [Q&A] [answered]
While I'm playing with solutions to these problems myself, I thought I'd reach out and see if anyone else had issues, and had a solution.  I'd first like to say I've been using Miryoku for most of work over the last 3 weeks so these are not knee jerk reactions, and I believe the switch is largely responsible for relieving some pain issues I've had with my right hand.

I really miss access to backspace in the number layer.  I find I need it much more here than in the navigation layer, but 0 is rather comfy in that location with the number pad.

I also really find it awkward for () to be on the thumb keys.  I'm a programmer and use these a lot.  This might be more specific to my use case having suffered a thumb injury many many years ago, and for reason this seems to irritate more than other uses of the thumb.  

Anyways, just curious if anyone has come up with similar challenges and has solutions they've been happy with over the long term.

**Comment by manna-harbour** [comment, marked as answer]
I'm glad it seems to be helping with your pain!

For backspace on Num, I just lift the thumb holding the Num layer and tap for backspace as usual. Combine with control to delete whole words. For auto-repeat you'd need to hold the opposite hand thumb for Nav, but that shouldn't be awkward.  After a while muscle memory will take over the multiple thumb functions.

The thumb parens are certainly odd at first but you should get used to them fairly quickly. In general, to optimise for your workflow, I suggest adding combos on Base for e.g. your most frequently used symbols or multi-symbol operators, particularly if they are made up of symbols from different layers. That way you can easily adjust the combos as your needs change, and you will always have the unmodified layers to fall back on.

Miryoku does make full use of the thumbs though, so if you have a thumb injury you might prefer to offload some functions to the fingers. For example, if you don't need to use every mod, the clipboard keys, or the Additional Functions while on Num, you could use https://github.com/manna-harbour/miryoku/discussions/85 to substitute a custom Num layer, and add e.g. backspace and parens on the fingers of the hand holding the Num layer thumb.
